code/citadel/organs/genitals.dm

```diff
@@ -198,21 +208,28 @@
 	if(H.is_groin_exposed(worn_stuff))
 		if(H.has_penis())
 			genitals_to_add += H.getorganslot("penis")
-		if(H.has_vagina())
-			genitals_to_add += H.getorganslot("vagina")
+		else
+			if(H.has_vagina()) //Sadly, a penis would cover the vagina and they use the same layer, so don't draw both.
```

Comment:
Yeah, I'm not sure what the order this would result in.
Do we WANT both to be drawn one over the other? I was a bit paranoid about drawings things being some sort of costly action after talking to TG coderbus about images...